app/views/home/_enrol_buttons.html.slim

```diff
@@ -1,4 +1,8 @@
 .govuk-button-group
-  = govuk_button_link_to 'Create an account', new_user_registration_path
-  .white-space-pre-wrap= ' or '
-  = govuk_link_to 'sign in', new_user_session_path
+  - if Rails.application.gov_one_login?
```

Comment:
@instantrick I believe we need to do something on `/about-training` to support the new `about` design and GovOne.